books.md

```diff
@@ -39,4 +47,5 @@
 * [The Boost C++ libraries](http://en.highscore.de/cpp/boost)
```

Comment:
may be the older site or ... but this always redirect me to "http://theboostcpplibraries.com/", why not use the target site url?